Chapters/0-1-Prologue.md

```diff
@@ -0,0 +1,49 @@
+
+The traditions say it's good luck to be born under a ziyu, and to be born under its song is a blessing unmatched by any other. I've never felt particularly blessed. I was born a normal maman in a farming town in Heiko province. As a child I tilled the soil and harvested root vegetables. I once caught the milta that had nibbled through at least 50 of our pohua crop, so I guess I'm quick as well, but that's pretty typical for a maman. I tried my luck as an apprentice at the observatory, but fate decided that it was my mother's time to go. Hae farm, and all of the responsibilities and expectations that hae community imposed upon it, had fallen upon me.
+
+Once, I fell in love with a man and a woman that I doted on in a drunken stupor and I was so afraid of my feelings that I joined with them as soon as I could so I would never have to fall in love again. As my luck would have it, they make my life easier for the most part, but occasionally I think that I'd prefer the company of a screaming toddler, though I won't get into that right now. These days, I leave my adventures to the dealings of my farm, and my library of books.
+
+But this day was special. It was my annual tradition with the ziyu – of returning to my place of birth. Though my parents had passed, I still felt their presence whenever I came back. They would welcome me as the tree did with its great branch-arms stretched wide toward Ara. "I'm back," I'd say, and they breathed a sigh of relief that I had made it another year. I'd settle in against its trunk and then its song would start to lift.
+
+But today, it was silent. -->
+
+<!--
+
+Scene: Thimlu has come to relieve Jathi of his observer duties
+
+- Jathi has found something
+- Thimlu sees it, is disheartened
+-
+
+
+-->
+
+"Oi, Jathi," He greeted his apprentice.
+
+"You're awake. Well that's odd. Usually you're under a blanket at the hearth by now."
+
+"Observer Thimlu!" Jathi exclaimed, abruptly standing up to greet him. His eyes were sharp with adrenaline, darkened by swelling under his eyes.
+
+"Don't tell me," said Thimlu. He paused to take a deep breath, then walked calmly to his desk and began to write a note.
+
+"How much?" he asked him.
+
+Jathi replied, hesitant, "10% coverage, sir. Last night it was–"
+
+"Four, yes I know," he said in a hurried tone.
+
+"Take this note to Melna before she leaves for the senate," said Thimlu.
+
+"Yes, sir," said Jathi. He took the note and turned to tidy the table he was at.
+
+"Leave the lenses," said Thimlu, "Go!"
+
+"Sorry," said Jathi. "Instinct."
+
+He tossed his coat on his back and opened the door to the summit of the observatory mountain. The morning breeze was on its first breath, making streaks in the fog below, and the water in his hydroderm swell. With a drink from the last of his water, he hurried down the mountain to the city below, carrying the weight of the world in his pocket.
```

Comment:
> With a drink from the last of his water

But he just left the place, why didn't he fil up water before heading out?

Reply:
og prologue renders this gone.